fixtures/tomshine/3-led-par-light-rgbuv.json

```diff
@@ -30,38 +30,74 @@
     }
   },
   "availableChannels": {
-    "Special Function": {
+    "No Function 1": {
+      "name": "No Function",
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "No Function 2": {
+      "name": "No Function",
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "Special Functions": {
       "defaultValue": 0,
       "capabilities": [
         {
           "dmxRange": [0, 10],
-          "type": "NoFunction"
+          "type": "NoFunction",
+          "switchChannels": {
+            "No Function / Color Presets": "No Function 1",
+            "No Function / Effect Speed / Strobe Speed": "No Function 2"
```

Comment:
Can you please rename `No Function / Effect Speed / Strobe Speed` to `Speed` to make it more concise and expressive?